components/admin_console/feature_discovery/feature_discovery.tsx

```diff
@@ -27,10 +28,13 @@ type Props = {
 
     imgPath: string;
 
+    prevTrialLicense: ClientLicense;
+
     stats?: Dictionary<number | AnalyticsRow[]>;
     actions: {
         requestTrialLicense: (users: number, termsAccepted: boolean, receiveEmailsAccepted: boolean, featureName: string) => Promise<{error?: string; data?: null}>;
         getLicenseConfig: () => void;
+        getPrevTrialLicense: any;
```

Comment:
Probably want this (trying to avoid the use of any when possible)
```suggestion
        getPrevTrialLicense: () => void;
```

Reply:
Done.